oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/meter/MeterSystem.java

```diff
@@ -187,6 +187,21 @@ public MeterSystem(final ModuleManager manager) {
             throw new IllegalArgumentException("Function " + functionName + " can't be found by javaassist.");
         }
         final String className = formatName(metricsName);
+
+        /**
+         * Check whether the metrics class is already defined or not
+         */
+        try {
+            CtClass alreadyMetric = classPool.get(METER_CLASS_PACKAGE + className);
+            if (alreadyMetric.getSuperclass() != parentClass || type != meterPrototypes.get(metricsName).getScopeType()) {
+                throw new IllegalArgumentException("Already defined metrics class " + metricsName
+                                                       + ", but calculate function or scope type is different.");
+            }
+            log.debug("Metric {} is already define, so skip the metric creation.", metricsName);
```

Comment:
```suggestion
            log.error("Metric {} is already define, so skip the metric creation.", metricsName);
```

Reply:
Such as the `OTEL` and `Zabbix` plugin, use the same metric name, I think it's a normal log. Why need to upgrade to the error level?

Reply:
I see your point. Keep it in INFO at least, this is important.